### ISSUE TITLE
Very long loading bootanimation

### ISSUE THREAD

**Issue by teerasak** [KoysX/treble_alpha_gsi#5]
exfat micro sdcard works, but vanilla variant take very long timed to boot-up from  start bootanimation. Thank you.

**Comment by roachsoft** [comment]
> exfat micro sdcard works, but vanilla variant take very long timed to boot-up from start bootanimation. Thank you.

Me too, in my s21U exynos.
In full package stay on bootanimation.

**Comment by Atman-5** [comment]
Yeah, there seem to be some issues with bootanimation on some devices, on some it starts on others it reboots or bootloops.
On v1.7.0 couldn't get past bootanimations even after deleting it, and to make it work had to try different vendors , all seemed okeish but for some reason battery draining is high, so i switched to v1.5.1
